### ISSUE TITLE
Improve game screen/menu

### ISSUE THREAD

**Issue by ExcaliburZero** [Infinite-Dango-Studios/ganbatte#24]
Currently the game has a built in sub-menu that appears while the game is being played. Unfortunately, the default sub-menu does not looks very good, and overlaps somewhat with the game screen.

The sub-menu can be seen at the bottom right of the following screenshot.

![screenshot0001](https://cloud.githubusercontent.com/assets/4276196/7030443/a799f24e-dd32-11e4-9a87-16640edb0069.png)

I have seen other Ren'Py based visual novels, such as Katawa Shoujo, that have opted to remove this sub-menu and replace it with a similar menu that pops up when the right mouse button is clicked.

An example of this can be found in the following screenshot.

![screenshot0001](https://cloud.githubusercontent.com/assets/4276196/7030502/1aaa3140-dd33-11e4-8e13-0ed806058858.png)

Since the pop-up sub-menu seems to look and work a bit better than the current default sub-menu, I think I will try to implement some sort of pop-up submenu.

**Comment by ExcaliburZero** [comment]
I've started working on the in-game menu using the add-game-menu-24 branch. It's going well so far.

Once I get some of the problems with it fixed I will merge it into the master branch.

**Comment by ExcaliburZero** [comment]
Here's a screenshot of the work in progress in-game menu:

![screenshot0003](https://cloud.githubusercontent.com/assets/4276196/7035476/f79fa0ee-dd56-11e4-9966-ceddd3c2fc5a.png)

**Comment by ExcaliburZero** [comment]
I've gotten the in-game menu working fine. However my implementation of it has messed up some other menus, as shown in the image below.

![screenshot0004](https://cloud.githubusercontent.com/assets/4276196/7037811/97295334-dd76-11e4-8676-8dccb4f9687c.png)

I think I'm going to try doing the menus in a method whereby they all pop-up over the current screen rather than being all on their won screen. I'm not sure, but I'll continue looking into the issue.

**Comment by ExcaliburZero** [comment]
I've gotten most of the problems with the menus fixed so far. I think I'll finish the odds and ends up, and then hold off on doing any more modifications to the menu system for a bit.

**Comment by ExcaliburZero** [comment]
I have gotten the menus all set for now, and have merged the branch I was working on into the master branch.

Since I have added in the in-game menu, I will close this issue.

**Comment by ExcaliburZero** [comment]
I have done a bit of work on getting an alternate method of implementing the in-game menu working. This method should be a bit more simplistic and work better.

**Comment by ExcaliburZero** [comment]
I think I've got the in-game menu working better now, using the new method of implementing it. I have now merge the changes I've made into the master branch.

Here is what it currently looks like:

![screenshot0005](https://cloud.githubusercontent.com/assets/4276196/7053127/f95d1d62-de00-11e4-8db8-3076a744eec8.png)

Since I've gotten the alternate implementation working, I will close this issue.
